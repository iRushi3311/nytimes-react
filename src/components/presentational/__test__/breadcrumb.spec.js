import React from 'react';
import renderer from 'react-test-renderer';
import BreadCrumb from '../breadcrumb.js';
import {MemoryRouter} from 'react-router-dom';

describe('>>>C O M P O N E N T S --- snapshot test for BreadCrumb', () => {

    let props,
        componentTree;

    beforeEach(() => {
        props = {
            backName: 'home',
            currentName: 'Search'
        };

        componentTree = renderer.create(
            <MemoryRouter>
                <BreadCrumb {...props}/>
            </MemoryRouter>
        ).toJSON();
    });

    it('+++ BreadCrumb match with snapshot ', () => {
        expect(componentTree).toMatchSnapshot();
    });

});