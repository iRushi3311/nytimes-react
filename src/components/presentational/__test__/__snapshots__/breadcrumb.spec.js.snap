// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>>C O M P O N E N T S --- snapshot test for BreadCrumb +++ BreadCrumb match with snapshot  1`] = `
<div
  className="sc-bdVaJa bhLhcS"
>
  <span
    className="sc-bwzfXH gCtkdg"
  >
    <a
      href="/"
      onClick={[Function]}
    >
       
      Home
       
    </a>
      
     &gt; 
      
    <a>
      Search
    </a>
  </span>
</div>
`;
